docs/sources/developers/plugins/add-distributed-tracing-for-backend-plugins.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Add distributed tracing for backend plugins
+---
+
+# Add distributed tracing for backend plugins
+
+> **Note:** This feature requires at least Grafana 9.5.0, and your plugin needs to be built at least with
+> grafana-plugins-sdk-go v0.157.0. If you run a plugin with tracing features on an older version of Grafana,
+> tracing will be disabled.
+
+## Introduction
+
+Distributed tracing allows backend plugin developers to create custom spans in their plugins, and send them to the same endpoint
+and with the same propagation format as the main Grafana instance. The tracing context is also propagated from the Grafana instance
+to the plugin, so the plugin's spans will be correlated to the correct trace.
+
+## Configuration
+
+> **Note:** Only OpenTelemetry is supported. If Grafana is configured to use a deprecated tracing system (Jaeger or OpenTracing),
+> tracing will be disabled in the plugin. Please note that OpenTelemetry + Jaeger propagator is supported.
+
+OpenTelemetry must be enabled and configured for the Grafana instance. Please refer to [this section](
+{{< relref "../../setup-grafana/configure-grafana/#tracingopentelemetry" >}}) for more information.
+
+As of Grafana 9.5.0, plugins tracing must be enabled manually on a per-plugin basis, by specifying `tracing = true` in the plugin's config section:
+
+```ini
+[plugin.myorg-myplugin-datasource]
+tracing = true
+```
+
+## Implementing tracing in your plugin
+
+> **Note:** Make sure you are using at least grafana-plugin-sdk-go v0.157.0. You can update with `go get -u github.com/grafana/grafana-plugin-sdk-go`.
+
+When OpenTelemetry tracing is enabled on the main Grafana instance and tracing is enabeld for a plugin,
+the OpenTelemetry endpoint address and propagation format will be passed to the plugin during startup,
+which will be used to configure a global tracer.
+
+<ol>
```

Comment:
Yeah I used HTML because I was having troubles making hugo correctly render orderer lists with code blocks inside. I agree it's not the best way to structure it, any further guidance on how to better divide those sections is very much appreciated! 😄

Reply:
What is the specific issue with ordered lists with code blocks inside? Is it that the numbers don't continue the sequence? If so, you can indent the block four spaces and this should solve the issue. See:  https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list